test/specs/scroll-transition.test.js

```diff
@@ -53,4 +53,92 @@ describe('<ScrollTransition />', () => {
+        expect(instance.getSlideStyles(1, 0).transform).toEqual(`scale(1)`);
+        expect(instance.getSlideStyles(2, 0).transform).toEqual(`scale(1)`);
+      });
+
+      it('should return zoomScale of 1 for slide index 0 and 0.85 as a default for slide index 1 & 2 when animation === zoom but no zoomScale defined ', () => {
+        const wrapper = mount(
+          <ScrollTransition animation="zoom">
+            <p>Slide 1</p>
+            <p>Slide 2</p>
+            <p>Slide 3</p>
+          </ScrollTransition>
+        );
+        const instance = wrapper.instance();
+        expect(instance.getSlideStyles(0, 0).transform).toEqual(`scale(1)`);
+        expect(instance.getSlideStyles(1, 0).transform).toEqual(`scale(0.85)`);
+        expect(instance.getSlideStyles(2, 0).transform).toEqual(`scale(0.85)`);
+      });
+
+      it('should return zoomScale of 1 for slide index 0 and zoomScale of 0.1 for index 1 & 2 when animation === zoom and zoomScale = 0.1 ', () => {
+        const wrapper = mount(
+          <ScrollTransition animation="zoom" zoomScale={0.1}>
+            <p>Slide 1</p>
+            <p>Slide 2</p>
+            <p>Slide 3</p>
+          </ScrollTransition>
+        );
+        const instance = wrapper.instance();
+        expect(instance.getSlideStyles(0, 0).transform).toEqual(`scale(1)`);
+        expect(instance.getSlideStyles(1, 0).transform).toEqual(`scale(0.1)`);
+        expect(instance.getSlideStyles(2, 0).transform).toEqual(`scale(0.1)`);
+      });
+
+      it('should return zoomScale of 1 for slide index 0 and 1 for the rest of the slides when animation === zoom and zoomScale value is over 1 ', () => {
+        const wrapper = mount(
+          <ScrollTransition animation="zoom" zoomScale={2.5}>
+            <p>Slide 1</p>
+            <p>Slide 2</p>
+            <p>Slide 3</p>
+          </ScrollTransition>
+        );
+        const instance = wrapper.instance();
+        expect(instance.getSlideStyles(0, 0).transform).toEqual(`scale(1)`);
+        expect(instance.getSlideStyles(1, 0).transform).toEqual(`scale(1)`);
+        expect(instance.getSlideStyles(2, 0).transform).toEqual(`scale(1)`);
+      });
+
+      it('should return zoomScale of 1 for slide index 0 and 0 for the rest of the slides when animation === zoom and zoomScale value is below 0 ', () => {
+        const wrapper = mount(
+          <ScrollTransition animation="zoom" zoomScale={-2.5}>
+            <p>Slide 1</p>
+            <p>Slide 2</p>
+            <p>Slide 3</p>
+          </ScrollTransition>
+        );
+        const instance = wrapper.instance();
+        expect(instance.getSlideStyles(0, 0).transform).toEqual(`scale(1)`);
+        expect(instance.getSlideStyles(1, 0).transform).toEqual(`scale(0)`);
+        expect(instance.getSlideStyles(2, 0).transform).toEqual(`scale(0)`);
+      });
+
+      it('should return zoomScale of 1 for all slides when animation !== zoom and zoomScale value is below 0 ', () => {
```

Comment:
What do you mean by `below 0` here? Are you referring to zoomScale not being passed in? or is the intention to check that the user is passing in a negative value?

Reply:
I was testing that the zoomScale value is not being passed when the animation is not set to "zoom". The positive/negative value does not make any difference in this case. I will change that. Good catch!

Reply:
Yeah, I think you can reword it to " ... and zoomScale is less than 0".

Reply:
Fixed.

Reply:
@johannam999 You're right, this test is just to show that zoomScale is ignored, so your it statement can be reworded to something like:
"should ignore zoomScale if animation !== zoom"

And then your test shows how it ignores the 0.5 zoomScale when all the slides are set to a scale of 1 😎

Reply:
Thanks I rephrased that.